src/ts/components/logo.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { Box } from '@blockstack/ui';
+import { BoxProps } from '@blockstack/ui/dist/box';
```

Comment:
This should be exported from the UI library I guess

Reply:
Meh, since it's only TS, I don't see it being a huge problem. But it wouldn't hurt to have it exported at the top-level.

Reply:
See PR here https://github.com/blockstackpbc/blockstack-ui/pull/18